src/Widgets/AppContainers/AbstractAppContainer.vala

```diff
@@ -212,7 +212,9 @@ namespace AppCenter {
                 action_button.amount = int.parse (this.package.get_suggested_amount ());
             }
 
-            action_stack.set_visible_child_name ("buttons");
+            if (action_stack.get_child_by_name ("buttons") != null) {
+                action_stack.set_visible_child_name ("buttons");
```

Comment:
can you change this to `action_stack.visible_child_name = "buttons";` when modifying this line